src/pages/view-test-result/components/vehicle-details-card/vehicle-details-card.ts

```diff
@@ -31,6 +35,18 @@ export class VehicleDetailsCardComponent {
     );
   }
 
+  public shouldHideDimensions() : boolean {
+    switch (this.category) {
+      case TestCategory.F:
+      case TestCategory.G:
+      case TestCategory.H:
+      case TestCategory.K:
```

Comment:
There is a ticket duplicated from this one in the first CI sprint which covers ADI 2 changes to the detailed test view, my assumption is that it will be addressed there.